azure-function-http/src/main/java/io/micronaut/azure/function/http/BinaryContentConfiguration.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2017-2023 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.azure.function.http;
+
+import io.micronaut.core.annotation.Internal;
+import jakarta.inject.Singleton;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Bean to check if response content is binary and should be base 64 encoded.
+ */
+@Internal
+@Singleton
+public final class BinaryContentConfiguration {
```

Comment:
We have this class in AWS as well. @timyates Can we move this class to servlet-core: 

https://github.com/micronaut-projects/micronaut-aws/blob/master/function-aws-api-proxy/src/main/java/io/micronaut/function/aws/proxy/BinaryContentConfiguration.java

Reply:
@timyates create an issue for this and add it to [5.1.0 Release project](https://github.com/orgs/micronaut-projects/projects/53/views/1).

Reply:
https://github.com/micronaut-projects/micronaut-servlet/pull/504